crates/holochain/src/core/ribosome/host_fn/accept_countersigning_preflight_request.rs

```diff
@@ -840,6 +839,19 @@ pub mod wasm_test {
             ..
         } = RibosomeTestFixture::new(TestWasm::CounterSigning).await;
 
+        // Run any arbitrary zome call for bob to force him to run init
+        let _: AgentActivity = conductor
+            .call(
+                &bob,
+                "get_agent_activity",
+                GetAgentActivityInput {
+                    agent_pubkey: bob_pubkey.clone(),
+                    chain_query_filter: ChainQueryFilter::new(),
+                    activity_request: ActivityRequest::Full,
+                },
+            )
+            .await;
+
```

Comment:
This is what's necessary to make the test pass (along with bumping up the countersigning session expiration time)